requirements.txt

```diff
@@ -1,4 +1,10 @@
-
+# Django/Framework Packages
+django>=1.4.12,<=1.4.22
+django-model-utils==2.3.1
+South>=0.7.6
+djangorestframework>=2.3.5,<=2.3.14
+pytz>=2012h
+pycrypto>=2.6
```

Comment:
Actually I mentioned this challenge to a few people and it doesn't seem like we have a best practice around library dependency sychronization.